tile-reduce/index.d.ts

```diff
@@ -0,0 +1,139 @@
+// Type definitions for tile-reduce 3.1.1
+// Project: https://github.com/mapbox/tile-reduce
+// Definitions by: Denis Carriere <https://github.com/DenisCarriere>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace NodeJS  {
+  interface Global {
+    mapOptions: any
+  }
+}
+
+interface startEvent {
+    (): void;
+}
+
+interface mapEvent {
+    (tile: TileReduce.Tile, workerId: number): void;
+}
+
+interface reduceEvent {
+    (result: any, tile: TileReduce.Tile): void;
+}
+
+interface endEvent {
```

Comment:
Yes this is a node-only library, it has to do with multiple workers over a distributed system. The `global.mapOption` isn't coming from any imports of `tile-reduce`.

Source code: 

https://github.com/mapbox/tile-reduce/blob/master/src/worker.js#L9

Documentation:
```javascript
// map.js
module.exports = function (sources, tile, write, done) {
  global.mapOptions.bufferSize; // = 4
};
```